lib/OpenQA/WebAPI/Plugin/Gru.pm

```diff
@@ -159,13 +159,13 @@ sub enqueue_limit_assets {
 }
 
 sub enqueue_download_jobs {
-    my ($self, $downloads, $job_ids) = @_;
-    return unless (%$downloads and @$job_ids);
+    my ($self, $downloads) = @_;
```

Comment:
Maybe we should still accept `$job_ids` here so Minion jobs which are still in the queue from before the update would still be processed. It would be used if `$block_job_ids` is `undef`.

Reply:
I am not sure if I understand correctly.  This function is called in `schedule_iso` and `create job`, and it will call `enqueue` to create the Minion jobs and add data into GruTasks and GruDependencies table. Will it affect the minion jobs which are still in the queue?